setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup
+import os.path
+import sys
+
+setup(
+    name = 'sphinx_fossasia_theme',
+    version = '0.0.1',
+    author = 'Ujjwal Bhardwaj',
+    author_email = 'ujjwalb1996@gmail.com',
+    url = 'https://github.com/fossasia/sphinx_fossasia_theme',
+    license = 'GNU',
+    description = 'A Sphinx theme specific to FOSSASIA\'s Projects',
+    packages = ['sphinx_fossasia_theme'],
+    include_package_data = True,
+    entry_points = {
+        'sphinx.html_themes': [
```

Comment:
Also It seems `sphinx.html_themes` entry point was added only in version 1.6. Before that the entry point was named `sphinx_themes`. http://www.sphinx-doc.org/en/stable/theming.html#distribute-your-theme-as-a-python-package

The theme may not work on sphinx <= 1.6